runtime/service_integ_test.go

```diff
@@ -94,6 +95,17 @@ func TestShimExitsUponContainerKill_Isolated(t *testing.T) {
 	)
 	require.NoError(t, err, "failed to create container %s", containerName)
 
+	_, err = client.NewContainer(testCtx,
+		fmt.Sprintf("should-fail-%s-%d", t.Name(), time.Now().UnixNano()),
+		containerd.WithRuntime(firecrackerRuntime, nil),
+		containerd.WithSnapshotter(naiveSnapshotterName),
+		containerd.WithNewSnapshot(snapshotName, image),
+		containerd.WithNewSpec(
+			oci.WithProcessArgs("sleep", fmt.Sprintf("%d", testTimeout/time.Second)),
+		),
+	)
+	require.Error(t, err)
```

Comment:
`require.Error(t, err, "should not be able to create additional container when no drives are available")`